happytransformer/adaptors/__init__.py

```diff
@@ -0,0 +1,16 @@
+from .adaptor import Adaptor
+from .albert import AlbertAdaptor
+from .berts import BertAdaptor,DistilBertAdaptor,RobertaAdaptor
+
+ADAPTORS = {
+    'BERT':BertAdaptor(),
+    'DISTILBERT':DistilBertAdaptor(),
+    'ROBERTA':RobertaAdaptor(),
+    'ALBERT':AlbertAdaptor()
+}
+
+def get_adaptor(model_type:str)->Adaptor:
```

Comment:
Are you sure this goes in init.py along with AdAPTORS? Or should init.py just reference it?

Reply:
It's a tiny function I don't see any reason it wouldn't live in here. I can move it if you want though

Reply:
It's okay, we can keep it here.